resources/views/device/tabs/latency.blade.php

```diff
@@ -0,0 +1,142 @@
+@extends('device.index')
+
+@section('tab')
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <form method="post" role="form" id="map" class="form-inline">
+                @csrf
+                <div style="position: relative">
+                    <div class="form-group">
+                        <label for="dtpickerfrom">From</label>
+                        <input type="text" class="form-control" id="dtpickerfrom" name="dtpickerfrom" maxlength="16"
+                               value="{{ $data['dtpickerfrom'] }}" data-date-format="YYYY-MM-DD HH:mm">
+                    </div>
+                    <div class="form-group">
+                        <label for="dtpickerto">To</label>
+                        <input type="text" class="form-control" id="dtpickerto" name="dtpickerto" maxlength=16
+                               value="{{ $data['dtpickerto'] }} " data-date-format="YYYY-MM-DD HH:mm">
+                    </div>
+                    <input type="submit" class="btn btn-default" id="submit" value="Update">
+                </div>
```

Comment:
Wonder if we should move these form into blade components? Would be nice since we use them on a lot of pages.

Reply:
Probably, blade components are a lot better in Laravel 7.  Sometimes I think it is better to add them after they are used a lot and go back and replace.  That way we don't have something that is meant to be reusable only used in on spot. It also gives you a little perspective to cover all the requirements.